esp/scm/ws_dfu.ecm

```diff
@@ -812,7 +821,43 @@ ESPresponse [exceptions_inline, nil_remove] EraseHistoryResponse
     ESParray<ESPStruct History, Origin> History;
 };
 
+ESPStruct DFUPartLocations
+{
+    int LocationIndex;
+    string Location;
+};
+
+ESPStruct DFUPartCopies
+{
+    int PartId;
+    ESParray<string> LocationIndexes;
+};
+
+ESPrequest DFUReadAccessRequest
+{
+    string Name;
+    string Cluster;
+    int ExpirySeconds(60);
+    ESPenum SecAccessType AccessType;
+    string JobId;
+    bool ReturnFileInfo(false);
+    bool ReturnJsonTypeInfo(false);
+    bool ReturnBinTypeInfo(false);
+};
 
+ESPresponse [exceptions_inline] DFUReadAccessResponse
+{
+    string MetaInfoBlob;
+    string KeyName;         // name of key used to encrypt
+    
```

Comment:
@wangkx @rpastrana NB: removed expiryMinutes from response. *expiryTime* is in the encoded blob in the reply for dafilesrv's eyes only.

Reply:
seems reasonable. The client will need to refresh upon timeout failure from DAFILESERV